base/abstractarray.jl

```diff
@@ -361,6 +361,8 @@ See also [`checkindex`](@ref).
 
 # Examples
 ```jldoctest
+julia> using Random
```

Comment:
This (and most other doc examples) don't actually need to use `rand` so we should replace those calls with something like `fill` instead since random just poisons the example (and even more now with a required `using Random`.

Reply:
Thanks for your review! Actually with the new change (`rand` is in `Base`), `using Random` isn't necessary anymore.